offlineimap/accounts.py

```diff
@@ -273,6 +273,17 @@ def syncrunner(self):
             if e.severity >= OfflineImapError.ERROR.CRITICAL:
                 raise
             return
+        
```

Comment:
whitespace errors.

Reply:
Strange. `git diff --check` didn't point this out.